source/driver/Driver.cpp

```diff
@@ -401,8 +427,26 @@ bool Driver::runPreprocessor(bool includeComments, bool includeDirectives) {
     output.setIncludeComments(includeComments);
     output.setIncludeDirectives(includeDirectives);
 
+    std::unordered_map<std::string, std::string> translations;
+
     while (true) {
         Token token = preprocessor.next();
+        if (fuzzIds && token.kind == TokenKind::Identifier) {
+            std::string name = std::string(token.valueText());
+            auto translation = translations.find(name);
+            if (translation == translations.end()) {
+                auto new_name = generateRandomAlphanumericString();
+                auto ret = translations.insert({name, new_name});
```

Comment:
I don't see how this insert could fail, you only enter this branch if the name is not in the map already.

Reply:
True, removed the check. Only putted there in order to completely use the result of the insert function.